prog/dftb+/lib_reks/reksgrad.F90

```diff
@@ -2426,6 +2423,7 @@ subroutine getRmatSp(env, denseDesc, neighbourList, nNeighbourSK, &
         real(dp), allocatable :: tmpMat(:,:)
         integer :: iAt1, iAt2, nOrb, LmaxR, mu, nu, iL
 
+        RmatSpL(:,:) = 0.0_dp
```

Comment:
Why is this additional zeroing needed? I think that modification from intent out to inout is enough in this routine.